README.rst

```diff
@@ -14,11 +14,11 @@ Transformer Engine (TE) is a library for accelerating Transformer models on NVID
 using 8-bit floating point (FP8) precision on Hopper GPUs, to provide better performance with lower
 memory utilization in both training and inference. TE provides a collection of highly optimized
 building blocks for popular Transformer architectures and an automatic mixed precision-like API that
-can be used seamlessly with your PyTorch code. TE also includes a framework agnostic C++ API that
-can be integrated with other deep learning libraries to enable FP8 support for Transformers.
+can be used seamlessly with your own framework-specific code. TE also includes a framework agnostic 
+C++ API that can be integrated with other deep learning libraries to enable FP8 support for Transformers.
```

Comment:
A general comment, there is no links to H100 fp8 in the document.
If by chance you know one, adding it would be great. If you don't know one, we should ask on slack.

Reply:
Do you mean blog posts etc about FP8?

Reply:
Any link would be better then fp8. Blog post would be great.

Reply:
We can have a discussion internally to find out the best link for this, then submit another PR for fix?

Reply:
Yeah, I think this could be done in another PR. We will have more stuff after GTC since there are a few talks about FP8 there.